src/ethernet_interface.cpp

```diff
@@ -1185,6 +1185,12 @@ std::string EthernetInterface::macAddress([[maybe_unused]] std::string value)
         elog<InvalidArgument>(Argument::ARGUMENT_NAME("MACAddress"),
                               Argument::ARGUMENT_VALUE(value.c_str()));
     }
+    catch (const std::exception&)
```

Comment:
Please test and make sure this catch section does not hit for invalid argument exception otherwise changes looks good